templates/song_rater/song_list.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Song_List</title>
+</head>
+<body>
+<h1>List of all Songs</h1>
+<table border="1" width="600">
+<tr>
+<td>Title</td>
+<td>Author</td>
+</tr>
+
+{% for song in object_list%}
```

Comment:
To increase the readability of this for loop, you could set a `context_object_name` attribute on your `ListView` class to set a custom name for the `object_list` context variable. I think that something like `song_list` would be nice. It doesn't really matter in this instance, as it's small, straightforward template, but it could help in more complicated situations.